tests/test_content_loader.py

```diff
@@ -0,0 +1,304 @@
+# coding=utf-8
+
+import unittest
+import mock
+
+import os
+import tempfile
+import yaml
+
+from dmutils.content_loader import ContentLoader
+
+
+def get_mocked_yaml_reader(mocked_content={}):
+    def read(yaml_file):
+        return yaml.load(mocked_content[yaml_file])
+    return read
+
+
+class TestContentLoader(unittest.TestCase):
+
+    @mock.patch("dmutils.content_loader.ContentLoader._yaml_file_exists")
+    @mock.patch(
+        "dmutils.content_loader.ContentLoader._read_yaml_file",
+        side_effect=get_mocked_yaml_reader({
```

Comment:
It looks good either way, but since you have a reference to the `mocked_read_yaml_file` object in test arguments you could assign the `side_effect` there too. That way I think you can move the now repeating `@mock.patch` decorators to wrap the whole class instead of repeating them for every test method, which as far as I remember should work the same way (class decorator will decorate each test method).